### ISSUE TITLE
The Hive importing wrong data 

### ISSUE THREAD

**Issue by nino-filigran** [OpenCTI-Platform/connectors#2825]
## Description

In [this ticket](https://github.com/OpenCTI-Platform/opencti/issues/8409) a user has an issue with the Hive, importing "corrupted" data into its platform, causing it to crash

**Comment by richard-julien** [comment]
Please @nino-filigran provides the detail of the problem on this ticket and close the other one.

**Comment by nino-filigran** [comment]
FYI regarding the HIVE issue @nino-filigran Over TheHive, I ingest emails and attachments inside these emails as attachments for the cases that we create over TheHive, also I have some connectors that may produce outputs that are unstructured/corrupted to OpenCTI. When I first integrated TheHive I received the error I attached before and in some cases, it didn't create an incident for it over OpenCTI and skipped them since observables are none type.

After that I started to have some problems in OpenCTI and tool was unstable and extremely slow. I fixed that by rebooting until it became completely unstable and OpenCTI container doesn't last but a few moments right now. So, my guess is that there are not enough restrictions on the received data from TheHive side and it kept on adding unstructured data to OpenCTI which billed up to have the tool completely unstable like what I'm having at the moment.
